README.md

```diff
@@ -13,13 +13,11 @@ A more detailed guide on metrics is expected to be added to the OpenTelemetry Go
 The Dynatrace exporter is added and set up like this:
 
 ```go
-  opts := dynatrace.Options{}
-  if token, exists := os.LookupEnv("API_TOKEN"); exists {
-    opts.APIToken = token
-    opts.URL = os.Getenv("ENDPOINT")
-  }
-
-  exporter, err := dynatrace.NewExporter(opts)
+  // No options are required. By default, metrics will be exported to the
+  // OneAgent running locally on the host. If no OneAgent is running,
```

Comment:
Maybe rephrase to "By default, the exporter will attempt to export to the local OneAgent endpoint.". Otherwise it sounds as if there was no graceful handling if there is no OneAgent running.